ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -635,27 +655,47 @@ export default defineComponent({
         // Complete Filing
         localState.loading = true
         // Build filing to api specs
-        const apiData = await buildApiData()
+        const apiData: MhrTransferApiIF = await buildApiData()
         // Submit Transfer filing
         const mhrTransferFiling =
           await submitMhrTransfer(apiData, getMhrInformation.value.mhrNumber, localState.staffPayment)
+
         localState.loading = false
 
         if (!mhrTransferFiling.error) {
-          setUnsavedChanges(false)
-          const newItem: RegTableNewItemI = {
-            addedReg: mhrTransferFiling.documentId,
-            addedRegParent: getMhrInformation.value.mhrNumber,
-            addedRegSummary: null,
-            prevDraft: mhrTransferFiling.documentId || ''
+          // Affidavit Transfer has a different flow
+          if (isTransferToExecutorUnder25Will.value) {
+            TransAffidavit.affidavitCompleted = true
+            localState.loading = true
+            localState.validate = false
+            localState.isReviewMode = false
+
+            setUnsavedChanges(false)
+
+            // Pre-fill Transfer Gift Sale with current registration info
+            // WIP: for now use init draft to pre-fill the transfer
+            apiData.registrationType = ApiTransferTypes.SALE_OR_GIFT
+            await initDraftMhrInformation(apiData as MhrTransferApiIF)
```

Comment:
I think this works for now!
Keep in mind, we will need to do the same thing when opening from the table, so this may be updated in that future ticket.

Reply:
Sorry, to expand here, the trigger being the FROZEN status.

Reply:
Yes, I can update in the future once we know what value to pre-fill, as I suspect we don't want all the data (from affi) in the new transfer (sale).